service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/DiagnosticReportMapper.java

```diff
@@ -41,6 +41,9 @@
 @Slf4j
 public class DiagnosticReportMapper {
 
+    public static final String DUMMY_SPECIMEN_ID = "Specimen/DUMMY-SPECIMEN";
```

Comment:
1) This format "ResourceType/Id" is a reference not an ID
2) I'm pretty sure we can't use hardcoded references here. What happens when the patient record contains multiple diagnostic reports with no specimen?

Reply:
> This format "ResourceType/Id" is a reference not an ID

Updated now to just "DUMMY-SPECIMEN" and "DUMMY-OBSERVATION".

> What happens when the patient record contains multiple diagnostic reports with no specimen?

That's fine, this ID is only used in our code for the dummy/default Specimen generation (to keep the schema valid, and checks similar to this, https://github.com/nhsconnect/integration-adaptor-gp2gp/pull/239/files#diff-60fce59b547f21c0c6a2664153b1a9f2d3e2e3432d3eb8131b10ae658ed43fe6R299).
In the HL7/XML output, the ID of a dummy/default Specimen won't be "DUMMY-SPECIMEN", it'll be a randomly generated UUID, like the example below:
```
<component typeCode="COMP" contextConductionInd="true">
    <CompoundStatement classCode="CLUSTER" moodCode="EVN">
        <id root="3316531F-5705-424C-9E1A-EE694FB411B4"/>
        <code code="123038009" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="specimen (specimen)"/>
        <statusCode code="COMPLETE"/>
        <effectiveTime>
            <center nullFlavor="NI"/>
        </effectiveTime>
        <availabilityTime value="20120726134600"/>
        <specimen typeCode="SPC">
            <specimenRole classCode="SPEC">
                <id root="1141EEE6-EDE3-4358-8554-FFED0EE59284"/>
                <id root="2.16.840.1.113883.2.1.4.5.2" extension="DUMMY"/>
                <effectiveTime>
                    <center value="20120726134600"/>
                </effectiveTime>
                <specimenSpecimenMaterial determinerCode="INSTANCE" classCode="MAT">
                    <desc>UNKNOWN</desc>
                </specimenSpecimenMaterial>
            </specimenRole>
        </specimen>
```

So if we have multiple diagnostic reports with no specimen, they'll always have different UUIDs (no conflicts)